src/modules/lifecycle_management.json

```diff
@@ -326,6 +496,91 @@
                         "$ref": "#/components/schemas/LifecycleFinishedParameters"
                     }
                 }
+            },
+            "AppSession": {
+                "title": "AppSession",
+                "type": "object",
+                "required": [
+                    "app",
+                    "runtime"
+                ],
+                "properties": {
+                    "app": {
+                        "description": "Information about the application",
+                        "type": "object",
+                        "required": [
+                            "id",
+                            "url"
+                        ],
+                        "properties": {
+                            "id": {
+                                "description": "The identifier for the app",
+                                "type": "string"
+                            },
+                            "url": {
+                                "description": "The url that the app will be launched with",
+                                "type": "string"
+                            },
+                            "title": {
+                                "description": "The display title of the app",
+                                "type": "string"
+                            },
+                            "catalog": {
+                                "description": "The catalog id that this app's content metadata comes from",
+                                "type": "string"
+                            }
+                        }
+                    },
+                    "runtime": {
+                        "description": "Information about the runtime that the application is running in",
+                        "type": "object",
+                        "required": [
+                            "id"
+                        ],
+                        "properties": {
+                            "id": {
+                                "description": "The identifier for the runtime",
+                                "type": "string"
+                            }
+                        }
+                    },
+                    "launch": {
+                        "description": "Launch information for this app that will be loaded",
+                        "type": "object",
+                        "properties": {
+                            "intent": {
+                                "description": "The firebolt navigation intent that this app will be loaded with",
+                                "$ref": "https://meta.comcast.com/firebolt/intents#/definitions/NavigationIntent"
+                            },
+                            "secondScreen": {
+                                "description": "Any information about the app being launched from a second screen",
+                                "$ref": "#/components/schemas/SecondScreenEvent"
```

Comment:
Isn't this also in the Core SDK? We should move this into `firebolt-schemas` if so.

Reply:
Yeah planned to do that, think it could be a later refactor

Reply:
okay, just don't forget :)